Doc/tutorial/inputoutput.rst

```diff
@@ -358,8 +358,8 @@ To read a file's contents, call ``f.read(size)``, which reads some quantity of
 data and returns it as a string (in text mode) or bytes object (in binary mode).
 *size* is an optional numeric argument.  When *size* is omitted or negative, the
 entire contents of the file will be read and returned; it's your problem if the
-file is twice as large as your machine's memory. Otherwise, at most *size* bytes
-are read and returned.
+file is twice as large as your machine's memory. Otherwise, at most *size*
+characters (in text mode) or *size* bytes (in binary mode) are read and returned.
```

Comment:
```suggestion
characters as a string(in text mode) or *size* bytes as a byte object(in binary mode) are read and returned.
```
Maybe it would better because it returns a string instead of a list of characters.

Reply:
IMO it's not needed cause an earlier bit of the paragraph says the return type: 

> as a string (in text mode) or bytes object (in binary mode)

Reply:
Is it better to keep the consistency?:D